packages/cli/src/commands/new/csharp.ts

```diff
@@ -68,7 +69,7 @@ export function CSharp(_: IContext): CommandModule<{}, Args> {
           return true;
         });
     },
-    handler: async ({ name, template, ttk }) => {
+    handler: async ({ name, template, start, ttk }) => {
```

Comment:
NIT: Not related to this line, but this file is called "csharp". Consider consistency with naming (dotnet, vs csharp etc)

Reply:
@heyitsaamir  can you elaborate? The template folder and command name are both `csharp`.

Reply:
I meant, consider consistency for naming when it comes to dotnet. For eg, we aren't using nodejs and  typescript both. We're just using typescript. So similarly, should err just use one mane when it comes to this?

Reply:
ok I'm assuming you mean the `alias`, updated to just be `cs`.